### PR TITLE
[DataMigration] Update the description of command New-AzDataMigrationToSqlDb

### PR DESCRIPTION
**Pull request by yadongyaly**
## Description

This PR is going to update the description of command New-AzDataMigrationToSqlDb. 
If the customer does not have table in the target database, suggest them to use New-AzDataMigrationSqlServerSchema command to do schema migration firstly. 

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by yadongyaly** [comment]
trigger the build pipeline again.

**Comment by yadongyaly** [comment]
ChangeLog: update the description of command New-AzDataMigrationToSqlDb to inform the customers that they can use New-AzDataMigrationSqlServerSchema command to do schema migration.

**Review by wyunchi-ms — APPROVED** [review]

File: src/DataMigration/DataMigration.Autorest/readme.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ directive:
 
   - from: swagger-document
     where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{sqlDbInstanceName}/providers/Microsoft.DataMigration/databaseMigrations/{targetDbName}"].put
-    transform: $["description"] = "Create a new database migration to a given SQL Db."
+    transform: $["description"] = "Create a new database migration to a given SQL Db. This command can migrate data from the selected source database tables to the target database tables. If the target database have no table existing, please use New-AzDataMigrationSqlServerSchema command to migrate schema objects from source database to target databse. The link of New-AzDataMigrationSqlServerSchema is https://learn.microsoft.com/powershell/module/az.datamigration/new-azdatamigrationsqlserverschema"
```
Comment (YanaXu) [inline src/DataMigration/DataMigration.Autorest/readme.md:91]: Update the description to "Create a new database migration to a given SQL Db. This command can migrate data from the selected source database tables to the target database tables. If the target database have no table existing, please use [New-AzDataMigrationSqlServerSchema](https://learn.microsoft.com/powershell/module/az.datamigration/new-azdatamigrationsqlserverschema) command to migrate schema objects from source database to target database.".
And re-generate the docs.

Comment (YanaXu) [reply src/DataMigration/DataMigration.Autorest/readme.md:91]: The link format is : `[New-AzDataMigrationSqlServerSchema](https://learn.microsoft.com/powershell/module/az.datamigration/new-azdatamigrationsqlserverschema)`.

Comment (yadongyaly) [reply src/DataMigration/DataMigration.Autorest/readme.md]: Thanks @YanaXu , please review the latest iteration of this change

**Review by YanaXu — APPROVED** [review]

**Pull request merged**